packages/dashboard-message-bus/README.md

```diff
@@ -0,0 +1,39 @@
+# @truffle/dashboard-message-bus
+
+:warning: **This is a middleware package and should only be used when developing integrations for the truffle dashboard** :warning:
+
+The `@truffle/dashboard-message-bus` manages the communication between the Truffle Dashboard and any other Truffle components, such as the browser provider. It works by running two separate WebSocket servers. One server is for clients to connect to and send requests, while the other server is for listeners (dashboards) to connect to and receive requests.
```

Comment:
Did we decide whether to call this "browser provider" or "dashboard provider"?

Reply:
Woops, that should be "dashboard provider"